### ISSUE TITLE
Firmware not flashing

### ISSUE THREAD

**Issue by joshuajaharwood** [NebraLtd/helium-bobcat-px30#10]
Hi,

I've followed your instructions for flashing this firmware on the PX30 model, but it just boots up as normal.

I have:

* Flashed to an SD card
* Put in TF slot
* Inserted Ethernet cable
* Inserted power cable and turned on

It then just boots to the normal OS. Am I missing something?

Many thanks,
Josh

**Comment by WayneNebra** [comment]
This is the one gig version of Bobcat?

**Comment by arduino43** [comment]
I had the same issue with my PX30. I was never able to get it to boot from SD.

**Comment by joshuajaharwood** [comment]
> This is the one gig version of Bobcat?

Yes, model MINER3001.

**Comment by joshuajaharwood** [comment]
An update:

I decided to do some investigating using ADB:

Turns out my model is not PX30-based, but RK3566-based! The label on the bottom reads MINER3001, yet it's the 2GB model. I'll try running the other image and see if that works.

Some notes:

By opening the device, you can access a microUSB port labelled 'USB_OTG'. By holding the recovery button as you insert the power cable, you can access an ADB debug shell that you can use to query system information (e.g. the CPU with `cat /proc/cpuinfo` or memory with `cat /proc/meminfo`.

**Comment by arduino43** [comment]
So you mind posting a photo of the internal?

**Comment by joshuajaharwood** [comment]
> So you mind posting a photo of the internal?

Of course. Just a moment

**Comment by joshuajaharwood** [comment]
![IMG_2941](https://github.com/NebraLtd/helium-bobcat-px30/assets/1280942/04775dc7-15d0-49f5-af17-af791e680358)
![board_back](https://github.com/NebraLtd/helium-bobcat-px30/assets/1280942/74c24f66-aa52-42e9-ada7-70be47295fb7)

**Comment by joshuajaharwood** [comment]
Yeah, tried with the RK3566 image and it's still just booting straight to the stock OS.

**Comment by arduino43** [comment]
Thanks! Yeah, this is fairly different looking than my PX30

**Comment by arduino43** [comment]
I’ve tried every method on both versions and nothing.

rktool says it completes fine, just bricks the device. Using SD on PX30 doesn’t work.

I’ve tried at least 4 rk3566 boards all the same result, and two px30 boards. 

Hopefully you get somewhere with it! Keep us updated.

**Comment by WayneNebra** [comment]
> An update:
> 
> I decided to do some investigating using ADB:
> 
> Turns out my model is not PX30-based, but RK3566-based! The label on the bottom reads MINER3001, yet it's the 2GB model. I'll try running the other image and see if that works.
> 
> Some notes:
> 
> By opening the device, you can access a microUSB port labelled 'USB_OTG'. By holding the recovery button as you insert the power cable, you can access an ADB debug shell that you can use to query system information (e.g. the CPU with `cat /proc/cpuinfo` or memory with `cat /proc/meminfo`.

2 gig version doesn't work at the moment.

**Comment by itestings** [comment]
> > An update:
> > I decided to do some investigating using ADB:
> > Turns out my model is not PX30-based, but RK3566-based! The label on the bottom reads MINER3001, yet it's the 2GB model. I'll try running the other image and see if that works.
> > Some notes:
> > By opening the device, you can access a microUSB port labelled 'USB_OTG'. By holding the recovery button as you insert the power cable, you can access an ADB debug shell that you can use to query system information (e.g. the CPU with `cat /proc/cpuinfo` or memory with `cat /proc/meminfo`.
> 
> 2 gig version doesn't work at the moment.

Did try a look at Crankk method ? i just installed for test Crankk Image on one of my Bobcat 295 which is 2GB version base on RK3566 - and it is working but there are issues with their firmware, so i hope that you can take a look over their method and make proper image for Bobcat 2GB version based on RK3566.
